bom/application/pom.xml

```diff
@@ -21,10 +21,6 @@
         <jandex.version>2.4.2.Final</jandex.version>
         <resteasy.version>4.7.5.Final</resteasy.version>
         <opentracing.version>0.33.0</opentracing.version>
-        <opentracing-jaxrs.version>1.0.0</opentracing-jaxrs.version>
-        <opentracing-web-servlet-filter.version>0.4.1</opentracing-web-servlet-filter.version>
-        <opentracing-tracerresolver.version>0.1.8</opentracing-tracerresolver.version>
-        <opentracing-concurrent.version>0.4.0</opentracing-concurrent.version>
```

Comment:
How are these things handled? You included them all in SmallRye OpenTracing?

Reply:
Yes, they are now part of SmallRye OpenTracing. They are included via a transitive dependency to `io.smallrye:smallrye-opentracing-contrib`.

I wonder how should we move this? Users may be adding these dependencies directly into their projects. Should we leave them and add a migration to the new one?

Reply:
We have this mechanism that might be helpful: https://github.com/quarkusio/quarkus/pull/7830/files#diff-0f3d0d9d3be9d8561e9e26160e885a43d54491289664d9556908451479c2d4dfR63-R68

I'm not sure if the message is very user friendly though.

If not, let's just add a note in the migration guide.

Reply:
I'm fine if we just add a note in the migration guide.

Reply:
I'll let you do that then :).